static/js/tab_bar.js

```diff
@@ -32,12 +35,14 @@ function make_tab_data(filter) {
             tab_data.rendered_narrow_description = current_stream.rendered_description;
             tab_data.sub_count = get_sub_count(current_stream);
             tab_data.formatted_sub_count = get_formatted_sub_count(current_stream);
+            // the "title" here is the stream name ie a user defined string, therefore they should NOT be translated by i18n
+            tab_data.sub_count_tooltip_text = i18n.t("__count__ users are subscribed to #" , {count: tab_data.sub_count}) + tab_data.title;
```

Comment:
What is the write way to add a piece of text that should not be translated into a string that needs to be translated?
The way this currently is will break eg in languages that are read right to left.